src/pages/en/guides/cms/contentful.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Contentful & Astro
+description: How to connect your Contentful CMS to your Astro site.
+layout: ~/layouts/CmsGuideLayout.astro
```

Comment:
```suggestion
layout: ~/layouts/MainLayout.astro
```
I'm just going to use the main layout for now so that we can see a Netlify preview of this as we work, too.